.github/workflows/release-drafter.yml

```
@@ -0,0 +1,20 @@
+name: Release Drafter
+
+on:
+  pull_request:
+    types: [closed]
+    branches: ['trunk']
+
+permissions:
+  contents: write
+
+jobs:
+  update_release_draft:
+    if: github.event.pull_request.merged == true # Run only when a PR is merged.
```

Not sure if this would work

```suggestion
    if: github.event.pull_request.merged # Run only when a PR is merged.
```

---

It should work per [GitHub's Expressions docs](https://docs.github.com/en/actions/learn-github-actions/expressions), but it is unneeded so I'm removing it.